packages/modules/src/modules/banners/europeMomentLocalLanguage/components/headerImageSvg.tsx

```diff
@@ -0,0 +1,530 @@
+import React from 'react';
+
+export function HeaderImageSvgMobileSmall(): JSX.Element {
+    return (
+        <svg
+            width="274"
+            height="40"
+            viewBox="0 0 274 40"
```

Comment:
Am I right in thinking these svgs are the same but require different widths, heights and viewboxes? If so, could pass these in to one svg component as props instead

Reply:
Yes have changed to make a single reusable component, looks almost identical.